applications/rag/metadata.yaml

```diff
@@ -17,27 +17,15 @@ spec:
   content: {}
   interfaces:
     variables:
-      - name: goog_cm_deployment_name
-        description: The name of the deployment.
-        varType: string
```

Comment:
Why delete the deployment name?

Reply:
its there.. just formatted to alphabetical order